theseus/geometry/functional/lie_group.py

```diff
@@ -27,16 +27,24 @@ def UnaryOperatorFactory(module, op_name):
     op_autograd_fn = getattr(module, "_" + op_name + "_autograd_fn")
```

Comment:
Let's add some comments at the top of this class describing what it does.

Reply:
I added the comments in `taoshaf.functional.so3.inverse`.

Reply:
> Let's add some comments at the top of this class describing what it does.

I added the comments in [`taoshaf.functional.so3.inverse`](https://github.com/facebookresearch/theseus/blob/8f01664ba4a178169250b94062df4cc7e429b465/theseus/geometry/functional/lie_group.py#L34).